docker/README.contrib.md

```diff
@@ -0,0 +1,27 @@
+# Build from node-zwave-js git repo
+
+In order to build a docker image from the master branch of the node-zwave-js, nake a small modification to package.json. Remove the zwave-js package from the file, like so:
+
+```
+$ git diff
+diff --git a/package.json b/package.json
+index 268bdda..c577a40 100644
+--- a/package.json
++++ b/package.json
+@@ -117,8 +117,7 @@
+     "vue-prism-editor": "^1.2.2",
+     "vue-router": "^3.4.3",
+     "vuetify": "^2.3.10",
+-    "vuex": "^3.5.1",
+-    "zwave-js": "^5.5.0"
++    "vuex": "^3.5.1"
+   },
+   "devDependencies": {
+     "@babel/core": "^7.11.5",
+```
```

Comment:
Why this ? Also I would suggest to better explain that you should firstly clone both projects in a folder:

```
mkdir -p testing && cd testing
git clone https://github.com/zwave-js/node-zwave-js
git clone https://github.com/zwave-js/zwavejs2mqtt
wget https://raw.githubusercontent.com/zwave-js/zwavejs2mqtt/master/docker/Dockerfile.contrib
docker build -t zwavejs2mqtt .
```

Reply:
I removed the zwave-js dependency so that npm doesn't install it. Otherwise we would have to delete it before overwriting with the master branch version, right?

Reply:
Good idea about the prerequisites explanation!

Reply:
You could also try changing the `zwave-js` dependency to `"file:/path/to/zwave-js/package"` if you want to avoid the intial installation of zwave-js from npm.
The `zwave-js` workspace gets linked in itself by npm, so that **might** work.

Reply:
> You could also try changing the zwave-js dependency to "file:/path/to/zwave-js/package" if you want to avoid the intial installation of zwave-js from npm.

Didn't know that was actually possible, is it possible to run somthing like `npm install file:/path/to/zwave-js/package` ?

Reply:
This can be deleted with my latest reviews